### PR TITLE
Honor step size on the logarithmic scale.

### PR DESCRIPTION
**Pull request by manthey**
Added a test to make sure the step size was honored.

### CODE REVIEW

**Comment by seiyria** [comment]
Seems fine. I'm not sure if this needs to be documented since it should be expected behavior anyway. @rovolution?

File: js/bootstrap-slider.js [hunk 1]
```diff
@@ -231,7 +231,17 @@
 				toValue: function(percentage) {
 					var min = (this.options.min === 0) ? 0 : Math.log(this.options.min);
 					var max = Math.log(this.options.max);
-					return Math.exp(min + (max - min) * percentage / 100);
+					var value = Math.exp(min + (max - min) * percentage / 100);
+					value = this.options.min + Math.round((value - this.options.min) / this.options.step) * this.options.step;
```
Comment (rovolution) [inline js/bootstrap-slider.js]: you are initializing `value` on line 234 only to re-set its value on the following line

**Comment by rovolution** [comment]
@manthey thanks for the PR! Could you see my comment on the code, make the update, and tag me again?

Thanks!

File: js/bootstrap-slider.js (hunk 1 above)
Comment (manthey) [inline js/bootstrap-slider.js]: @rovolution Line 234 calculates the unstepped value.  Line 235 uses the value calculated in line 234 and rounds it to a stepped value.  It seemed clearer to have it in two lines than in one.

Comment (rovolution) [inline js/bootstrap-slider.js]: oh i see it now! thanks for clarifying that!

**Pull request merged**